<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-14S2SR6" failures="0" tests="3" name="com.nocommercedemo.Testcases.HomePageTest" time="41.716" errors="1" timestamp="2024-07-12T15:34:29 IST" skipped="0">
  <testcase classname="com.nocommercedemo.Testcases.HomePageTest" name="afterSuite" time="5.038">
    <error message="Cannot invoke &quot;com.relevantcodes.extentreports.ExtentReports.flush()&quot; because &quot;com.nocommercedemo.utility.ExtentManager.extent&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.relevantcodes.extentreports.ExtentReports.flush()" because "com.nocommercedemo.utility.ExtentManager.extent" is null
at com.nocommercedemo.utility.ExtentManager.endReport(ExtentManager.java:75)
at com.nocommercedemo.Base.Baseclass.afterSuite(Baseclass.java:113)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- afterSuite -->
  <system-out/>
  <testcase classname="com.nocommercedemo.Testcases.HomePageTest" name="homepagepageTitle" time="3.467"/>
  <system-out/>
  <testcase classname="com.nocommercedemo.Testcases.HomePageTest" name="homepageallfunctions" time="33.211"/>
  <system-out/>
</testsuite> <!-- com.nocommercedemo.Testcases.HomePageTest -->
